src/mpyl/steps/deploy/k8s/resources/dagster.py

```diff
@@ -22,6 +22,20 @@ def to_grpc_server_entry(host: str, location_name: str, port: int) -> dict:
     return {"grpc_server": {"host": host, "location_name": location_name, "port": port}}
 
 
+def collect_extra_manifests(
+    builder: ChartBuilder, sealed_secrets_count: int, release_name: str
+) -> List[CustomResourceDefinition]:
+    sealed_secret_manifest = builder.to_sealed_secrets()
+    sealed_secret_manifest.metadata.name = release_name
+
+    extra_manifests_definitions = list(to_prometheus_chart(builder).values())
+
+    if sealed_secrets_count > 0:
+        extra_manifests_definitions.append(sealed_secret_manifest)
+
+    return extra_manifests_definitions
```

Comment:
if the need arises to deploy an extraManifest for the third time, i will take some time to see how we can generalize this a bit better?